### PR TITLE
Add switch-libwebp

### PR DESCRIPTION
**Pull request by uyjulian**

### CODE REVIEW

File: switch/libwebp/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+options=(!strip libtool staticlibs)
+makedepends=('devkitpro-pkgbuild-helpers')
+source=("https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-$pkgver.tar.gz")
+sha256sums=('3d47b48c40ed6476e8047b2ddb81d93835e0ca1b8d3e8c679afbb3004dd564b1')
+
+build() {
+  cd libwebp-$pkgver
+
+  source /opt/devkitpro/devkita64.sh
+  source /opt/devkitpro/switchvars.sh
```
Comment (WinterMute) [inline switch/libwebp/PKGBUILD]: Again, as per comments on https://github.com/devkitPro/pacman-packages/pull/88 only switchvars.sh is required

File: switch/libwebp/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  CFLAGS="${CFLAGS} -D__SWITCH__ " \
```
Comment (WinterMute) [inline switch/libwebp/PKGBUILD]: and again, CPPFLAGS sets -D__SWITCH__ check if the library uses CPPFLAGS when configuring.

**Comment by uyjulian** [comment]
Changes have been implemented and pushed

**Comment by WinterMute** [comment]
Thanks for your contribution. Much appreciated

**Pull request merged**